index.js

```diff
@@ -94,21 +110,20 @@ const createElement = (tagType, attrObj, parentElement) => {
     return newElement
 }
 
-const createGlobalWrapper = () => {
-    createElement("div", {
-        id: ELEMENT_IDS.wrapper
-    }, appElement)
-}
-
 const handleOptionsElementOnClick = (tabName) => {
     const allTabIds = Object.entries(ELEMENT_IDS)
         .filter(entry => entry[0].endsWith('Tab'))
         .map(entry => entry[1])
     const tabNameTabButtonIdValue = ELEMENT_IDS[`${tabName}Tab`]
 
-    allTabIds.forEach((tabId) => {
-        document.getElementById(tabId).disabled = tabId === tabNameTabButtonIdValue
-    })
+    allTabIds.forEach((tabId) => document.getElementById(tabId).disabled = tabId === tabNameTabButtonIdValue)
+}
+
+const removeSubMenuWrapperElement = (element) => {
+    element = document.getElementById(ELEMENT_IDS.subMenuWrapper)
+    if (element) {
+        element.remove()
```

Comment:
Do you know `?.` syntax? If not, you could research on it and apply it here instead if

Reply:
It is called optional chaining

Reply:
I read after and tried, left you a comment.